tests/acceptance/features/bootstrap/FeatureContext.php

```diff
@@ -1748,7 +1490,20 @@ public function theHTTPStatusCodeShouldBeBetween(
 	 * @return void
 	 */
 	public function theHTTPStatusCodeShouldBeSuccess(): void {
-		$this->theHTTPStatusCodeShouldBeBetween(200, 299);
+		$minStatusCode = 200;
+		$maxStatusCode = 299;
+		$statusCode = $this->response->getStatusCode();
+		$message = "The HTTP status code $statusCode is not between $minStatusCode and $maxStatusCode";
+		Assert::assertGreaterThanOrEqual(
+			$minStatusCode,
+			$statusCode,
+			$message
+		);
+		Assert::assertLessThanOrEqual(
+			$maxStatusCode,
+			$statusCode,
+			$message
+		);
```

Comment:
Why do we need this if the same thing is done in `theHTTPStatusCodeShouldBeBetween`?
IMO, let's remove step implementation of `the HTTP status code should be success`. The method `theHTTPStatusCodeShouldBeSuccess` is used only once in `Sharing.php`, it can be refactored using `theHTTPStatusCodeShouldBeBetween` method.